invadedhouse/commands/drop.py

```diff
@@ -0,0 +1,34 @@
+from commands import Command
+
+
+class Drop(Command):
+
+    def do(self, text: str):
+        player = self.get_player()
+        current_room = player.get_location()
+        item_str = text
+
+        if item_str in ('all', 'everything', 'all my belongings', 'everything i own'):
+            count = 0
+            for item in player.all_items():
+                count += 1
+                player.drop(item)
+                current_room.add_item(item)
+                dots = '.' * count
+                print(f'Dropped {item.get_name()}{dots}')
```

Comment:
hehe I add an extra dot for each item that you're dropping when you're dropping everything. try this out when you have a lot of stuff in your inventory, it's fun.

Reply:
oh then if we make that pee command and you just do it a bunch of times, and then if we have the option to say 'loot floor', wen you drop all it'll just be like  you dropped pee. you dropped pee.. you dropped pee... you dropped pee …. you dropped pee..... and then the end is like you dropped pee...……………………………………………….………………………………………………………..